docs/releases.rst

```diff
@@ -19,6 +19,9 @@ running discovered tests on the old system or new system, respectively.
 The ``epel`` and ``fips`` :ref:`/plugins/prepare/feature` plugins
 were fixed to work well against ``RHEL-8`` hosts.
 
+The :ref:`/plugins/provision/beaker` provision plugin gains support for
+bootc installation, with user-provided ready-to-use images.
+
 
```

Comment:
I would mention this MR, I believe the feature is definitely user facing, greatly improving the experience:

https://github.com/teemtee/tmt/pull/3941

Reply:
And also https://github.com/teemtee/tmt/pull/3997/files

Reply:
And also https://github.com/teemtee/tmt/pull/3979

Reply:
Added in bd618f55: #3941, #3979, #3997.